apps/web/src/components/CallToActionCard/CallToActionCard.tsx

```diff
@@ -24,8 +24,8 @@ const CallToActionCard = ({
   ...rest
 }: CallToActionCardProps) => (
   <div
-    data-h2-background-color="base(white) base:dark(black.light)"
-    data-h2-color="base(black) base:dark(white)"
+    data-h2-background-color="base(foreground)"
```

Comment:
So `background` and `foreground` actually refer to whether an object should be a part of the page background, or whether it should be a foreground element. They're both specific to `background-color`.

Reply:
Okay, I get what it is referring to now. Though, it still seems kind of confusing to be setting the `background` attribute to something named `foreground` 😅 

I'm wondering if we could use hydrogen modifiers to control that instead 🤔 Maybe something like layers/elevation/z-index? Where the increasing value darkens in light mode and lightens in light mode. We could then pair it with setting text to `foreground.X` and do the inverse.

```html
<!-- Multiple values -->
<div
  data-h2-background-color="base(background)"
  data-h2-background-color="base(background.2)"

  data-h2-color="base(foreground)"
  data-h2-color="base(foreground.2)"
/>

<!-- Or, single value -->
<div
  data-h2-background-color="base(background)"
  data-h2-background-color="base(background.elevated)"

  data-h2-color="base(foreground)"
  data-h2-color="base(foreground.elevated)"
/>
```

The exact approach would benefit from some more thought than I have given it. Ie. The numbers could be confusing, since they are already being used for opacity and `elevated` makes me think it should also have shadows 😅

Reply:
I spun this off into https://github.com/GCTC-NTGC/gc-digital-talent/issues/8413